Sources/AppStoreConnectCLI/Helpers/File+Helper.swift

```diff
@@ -0,0 +1,39 @@
+// Copyright 2020 Itty Bitty Apps Pty Ltd
+
+import Foundation
+import Files
+
+extension File {
+    static func createFile(
+        in folderName: String,
+        named fileName: String,
+        with content: String
+    ) throws -> File {
+        let folder = try Folder(path: folderName)
+
+        return try folder.createFile(
+            named: fileName,
+            contents: Data(base64Encoded: content)
+        )
+    }
+
+    static func getFolderAndFileName(from path: String) -> (String, String) {
```

Comment:
```suggestion
    static func folderAndFilename(from path: String) -> (String, String) {
```

Though most of this function is unnecessary. You should probably just be using `URL` apis.

Eg:
```
        let f = URL(fileURLWithPath: "test/test")
        f.deletingLastPathComponent().path
        f.lastPathComponent
```

Though URL has issues with relative paths in the format of "./test".  You get `/` as the folder which is wrong.

So you can use `NSString` APIs:

```
let f: NSString = "./test"
let f2 = f.standardizingPath as NSString

f2.lastPathComponent
f2.deletingLastPathComponent
```
`standardizingPath` will remove any `../.././` stuff from the path.

So if `deletingLastPathComponent` returns an empty string then you know that the path is relative.

Unfortunately Apple hasn't put any of these path manipulation APIs on `String` in Swift.  I guess they consider them deprecated or not ideal so aren't exposing them on `String`.

They've keep the URL APIs.  But relative paths and file:// URLs are problematic.